server/src/internalClusterTest/java/org/elasticsearch/search/basic/SearchWithRandomIOExceptionsIT.java

```diff
@@ -171,21 +171,21 @@ public void testRandomDirectoryIOExceptions() throws IOException, InterruptedExc
                     .setQuery(QueryBuilders.matchQuery("test", English.intToEnglish(docToQuery)))
                     .setSize(expectedResults).get();
                 logger.info("Successful shards: [{}]  numShards: [{}]", searchResponse.getSuccessfulShards(), numShards.numPrimaries);
-                if (searchResponse.getSuccessfulShards() == numShards.numPrimaries && !refreshFailed) {
+                if (searchResponse.getSuccessfulShards() == numShards.numPrimaries && refreshFailed == false) {
                     assertResultsAndLogOnFailure(expectedResults, searchResponse);
                 }
                 // check match all
                 searchResponse = client().prepareSearch().setQuery(QueryBuilders.matchAllQuery())
                     .setSize(numCreated + numInitialDocs).addSort("_uid", SortOrder.ASC).get();
                 logger.info("Match all Successful shards: [{}]  numShards: [{}]", searchResponse.getSuccessfulShards(),
                         numShards.numPrimaries);
-                if (searchResponse.getSuccessfulShards() == numShards.numPrimaries && !refreshFailed) {
+                if (searchResponse.getSuccessfulShards() == numShards.numPrimaries && refreshFailed == false) {
                     assertResultsAndLogOnFailure(numCreated + numInitialDocs, searchResponse);
                 }
             } catch (SearchPhaseExecutionException ex) {
                 logger.info("SearchPhaseException: [{}]", ex.getMessage());
                 // if a scheduled refresh or flush fails all shards we see all shards failed here
-                if (!(expectAllShardsFailed || refreshResponse.getSuccessfulShards() == 0 ||
+                if (false == (expectAllShardsFailed || refreshResponse.getSuccessfulShards() == 0 ||
```

Comment:
can we enforce having `XXX == false` instead of `false == XXX` somehow?

Reply:
I'd simplify this if statement and keep XXX == false here as anywhere else

Reply:
Agreed, I don't know why I did it that way now.